lib/detect-testing-library-utils.ts

```diff
@@ -174,69 +181,6 @@ export function detectTestingLibraryUtils<
       return isNodeComingFromTestingLibrary(referenceNodeIdentifier);
     }
 
-    /**
-     * Determines whether a given node is a simulate event util related to
-     * Testing Library or not.
-     *
-     * In order to determine this, the node must match:
-     * - indicated simulate event name: fireEvent or userEvent
-     * - imported from valid Testing Library module (depends on Aggressive
-     * Reporting)
-     *
-     */
-    function isTestingLibrarySimulateEventUtil(
```

Comment:
I had to revert this one since it can't be reused for `user-event`: this is a default import coming from a different module.